bigbluebutton-html5/imports/ui/components/chat/component.jsx

```diff
@@ -53,4 +65,8 @@ export default class Chat extends Component {
       </section>
     );
   }
+
+  closeChat() {
```

Comment:
You should create a prop `handleClosePrivateChat` and pass down the `createClosedChatSession`.